jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/ExceptionMappers.java

```diff
@@ -16,6 +16,8 @@
 
 package com.palantir.remoting1.servers.jersey;
 
+import com.google.common.collect.ImmutableList;
+import java.util.List;
 import javax.ws.rs.ext.ExceptionMapper;
 
 public final class ExceptionMappers {
```

Comment:
worth just combining this class into the `JerseyServers` class? Can still keep the `getExceptionMappers` method, just consolidate classes?